tutorials/integrations/zapper.md

```diff
@@ -0,0 +1,105 @@
+
+You can follow these steps to review any of the aforementioned protocols:
+
+1. Select **Moonbeam** in the chain filter on the left sidebar. You'll be able to view the most trending DApps in the ecosystem and click on the protocol you want to explore 
+![Zapper DeFi Section](/images/tutorials/integrations/Zapper/zapper-4.webp)
+2. In the **Activity** tab, you'll be able to see the latest transactions for the selected protocol: token swaps, LP or staking on StellaSwap’s decentralized exchange, lending and borrowing transactions on Moonwell, staked GLMR on OmniLST and any other supported DApp on Zapper
+![StellaSwap on Zapper](/images/tutorials/integrations/Zapper/zapper-5.webp)
+3. You'll also be able to view other details of the selected protocol, such as the daily active users and the daily transactions in the left side-bar, all deployed smart contracts in the **Contracts** tab and your own holdings on the protocol by going to the **Properties** tab
+
+
+### Access Moonbeam Assets on Zapper
+
+Zapper also has an integrated modal for seamlessly swapping assets on Moonbeam. Users can access this feature by selecting **Moonbeam** in the chain filter on the left sidebar and choosing the desired token on the right sidebar.
+
+![Zapper Token Selection](/images/tutorials/integrations/Zapper/zapper-6.webp)
+
+Inside the token page, users will be able to see the asset's performance (price action, volumes, holders, and transactions). The swap modal will be accesible by clicking on the **Buy**/**Sell** buttons on the left sidebar. 
+
+![Zapper Swap Modal](/images/tutorials/integrations/Zapper/zapper-7.webp)
+
+Inside the swap modal you can select the assets for swapping, input the desired quantity for the token you want to sell, and click on **Review order** to sign and submit the transaction.
+
+![Zapper Swap Modal](/images/tutorials/integrations/Zapper/zapper-8.webp)
+
+## Integrate with Zapper's API
+
+Developers can leverage Zapper's API to enrich their applications with comprehensive DeFi data from the Moonbeam network.
+
+You can register for an API key by visiting the [Zapper API portal](https://protocol.zapper.xyz){target=\_blank}. An API key is necessary to authenticate your application and access Zapper's endpoints. 
+
+Zapper offers several endpoints to fetch relevant data, such as:
+
+- **Balances**: retrieve asset balances for addresses on Moonbeam
+- **Transactions**: access transaction histories
+- **Prices**: obtain real-time token prices
+
+Furthermore, you can familiarize yourself with the [Zapper API documentation](https://protocol.zapper.xyz/docs/api){target=\_blank}, which provides detailed information on the available endpoints, request structures, and response formats.
+
+Depending on your preferred programming language, you can set up your environment to make HTTP requests to Zapper's API. Below is an example using Node.js:
```

Comment:
```suggestion
Depending on your preferred programming language, you can set up your environment to make HTTP requests to Zapper's API. Below is an example using Node.js and [Axios](https://axios-http.com/docs/intro){target=\_blank}:
```

Reply:
Adding the link to Axios will help users get to the install instructions if they need them